__tests__/services/service.spec.ts

```diff
@@ -0,0 +1,149 @@
+import {
+    Status,
+    RemoteDataSuccess,
+    RemoteDataFailure
+} from '../../libs/remoteData';
+
+import { 
+    service,
+    applyDataTransformer,
+    applyErrorTransformer,
+    resolveServiceMap,
+    PromiseRemoteDataResultMap
+} from '../../services/service';
+
+describe('Service `service`', () => {
+
+    interface TransformedData<T> {
+        transformed: T;
+        isTransformed: true;
+    }
+
+    const responseDataFailed = <RemoteDataFailure<string>> {
+        error: 'error',
+        status: Status.Failure
+    }
+
+    const responseDataSuccess = <RemoteDataSuccess<string>> {
+        data: 'data',
+        status: Status.Success
+    }
+
+    const transformer = <T = any>(response: T): TransformedData<T> => {
+        return {
+            transformed: response,
+            isTransformed: true
+        }
+    }
+
+    describe('Method `service`', () => {
+        test('returns success response', async () => {
+            const result = <any>await service({ url: 'success' }) 
+
+            expect(result.status).toBe('Success')
```

Comment:
There is `isSuccess` method please use it here to avoid any type.
https://github.com/beda-software/aidbox-react/blob/master/libs/remoteData.ts#L59

P.S.
We need a better way than we use right now in tests
```typescript
expect(isSuccess(result)).toBeThruly();
if(isSuccess(result)) {
    expect(result.data).toBe('data-success')
}
```

Reply:
Maybe we should write own expect helpers for testing, something like this:
except(result).toBeSuccess(data);
except(result).toBeFailure(error);

They can be re-used in other projects.

Reply:
@ufocoder please add proposed expect helpers.